src/utils/fromRoot.js

```diff
@@ -1,7 +1,9 @@
 var _ = require('lodash');
-var dirname = require('path').dirname;
-var join = require('path').join;
-var normalize = require('path').normalize;
 var root = require('./packageJson').__dirname;
+var {
```

Comment:
Bleh. You and your multi-linedness. What do you think about a rule about keeping destructuring on 1 line?